sphinx/writers/latex.py

```diff
@@ -727,6 +733,10 @@ def hypertarget(self, id, withdoc=True, anchor=True):
         return (anchor and '\\phantomsection' or '') + \
             '\\label{%s}' % self.idescape(id)
 
+    def hypertarget_to(self, node, anchor=False):
+        # type: (nodes.Node, bool) -> unicode
+        return ''.join(self.hypertarget(node_id, anchor=anchor) for node_id in node['ids'])
+
```

Comment:
What do you think about inserting the `\phantomsection` only once, in case `anchor` is not False. Something like

```
    if node.get('ids'):
        return (anchor and '\\phantomsection' or '') + \
            ''.join(self.hypertarget(node_id, anchor=False) for node_id in node['ids'])
    return ''
```

Reply:
Good idea! I'll improve this later.